src/SearchInput/SearchInput.tsx

```diff
@@ -93,33 +110,44 @@ export const SearchInput: FC<SearchInputProps> = ({
           {label}
         </Text>
       )}
-      <Input
-        id={id}
-        type="text"
-        name={name}
-        placeholder={placeholder}
-        autoComplete="off"
-        value={selectedResult}
-        onKeyUp={search}
-        onChange={updateInputState}
-      />
-
-      <ResultsContainer
-        show={active}
-        absolutePosition={!resultsRelativePosition}
-      >
-        <ResultsList>
-          {list.length ? (
-            list.map((el, i) => (
-              <li key={i} onClick={() => select(el)}>
-                {el.label}
-              </li>
-            ))
-          ) : (
-            <li>No results</li>
-          )}
-        </ResultsList>
-      </ResultsContainer>
+
+      <InnerContainer hasBorder={hasBorder}>
+        <Input
+          id={id}
+          type="text"
+          name={name}
+          placeholder={placeholder}
+          autoComplete="off"
+          value={selectedResult}
+          onKeyUp={search}
+          onChange={updateInputState}
+          hasBorder={hasBorder}
+        />
+
+        {showCaret && (
+          <IconContainer>
+            <Icon render="caret-down" />
+          </IconContainer>
+        )}
+
```

Comment:
shouldn't you close Icon Container here?

Reply:
Not sure what you mean... it has a closing tag

Reply:
sorry I ment the InnerContainer. I don't think it should wrap the results. As far as I saw it the inner container should just be a wrapper for the input and the icon, right?

Reply:
Please add an explicit size prop declaration for the Icon 😎